projects/common/src/color/color-palette.ts

```diff
@@ -57,3 +50,14 @@ export class ColorPalette {
     return [...this.basisColors];
   }
 }
+
+export const getContrastingColor = (rgbColorString: string): string => {
```

Comment:
would suggest making the method public rather than a separate const function - unless there's more to this?

Reply:
Thats fine, it just necessitates there already being a color palette, and in my case it works because i have a color palette to use, however it seems like a special case. Normally i would think the consumer doesn't have a color palette

Reply:
Ah - I missed this was color palette, assumed it was color service. I would probably put it there eventually - eventually figuring out how to get it in here (such as by providing a color service on construction).

Reply:
kk, i'll leave here for now then

Reply:
Removed exported, and made function public